Tools/autotest/arducopter.py

```diff
@@ -1086,6 +1086,42 @@ def test_vibration_failsafe(self):
         self.context_pop()
         self.reboot_sitl()
 
+    def test_motor_failsafe_mode(self, mode, user_takeoff=False):
+        # stabilize check
+        self.progress("Motor failsafe check in %s" % mode)
```

Comment:
I think we should be consistent with the naming, so let's use "takeoff check" instead of "motor failsafe".

Reply:
Fixed